### DISCUSSION TITLE
Analog Drone Present

### DISCUSSION THREAD

**Discussion by Maincrosift** [ntsc-rs/ntsc-rs#189] [Presets]
![Philips_PM5544 svg_ntsc](https://github.com/user-attachments/assets/44e5e6b2-c7b2-44ed-981f-f65d71499f47)
![SMPTE_Color_Bars svg_ntsc](https://github.com/user-attachments/assets/222859d9-ad5a-417f-a423-39a8382488fc)
![florst_ntsc](https://github.com/user-attachments/assets/fed1b81e-6eda-44d7-8b93-e82b61730fc7)

This is a present of those analog drones. this is my first time doing presents like this, i tried my best.

For the "head switching" and "Tracking noise", you may need to adjust it's position because it may move depending of the video's resolution, you can also remove them if you want too.

[analog drone.json](https://github.com/user-attachments/files/17121417/analog.drone.json)

You can use it as you please. You don't need to credit (but if you do i will be pretty happy) but don't steal it and say that you've made it.

**Comment by Degamisu** [comment]
Good work! I like the small artifacts and details here. One thing to note, the 'Head switching' artifact would not be suitable here, as it is for recorded tapes! Unless it is meant to be on a reel/tape, I don't see the need for the Tracking Noise & Head Switching Noise. Excellent work though!

**Reply by Maincrosift** [reply]
Thank you. The head switch is in there because some of the images I've seen for reference (specially in a page) had that.
